server/src/api.js

```diff
@@ -0,0 +1,30 @@
+"use strict";
+
+const express = require('express')
+
+const router = express.Router([])
+const TransactionCtrl = require('./lib/db/transaction.ctrl')
+const ChangellyCtrl = require('./lib/changelly/changelly.ctrl')
+const OrderCtrl = require('./lib/order/order.ctrl')
+const EtaCtrl = require('./lib/order/eta.ctrl')
+const OrderStatusCtrl = require('./lib/order/status.ctrl')
+
+// TODO: Matt -- What are theses? Do they actually doing anything?
+router.get('/db/transaction/:id', TransactionCtrl.getTransaction)
+router.get('/db/transaction', TransactionCtrl.getTransaction)
+
```

Comment:
These are just for testing iirc

Reply:
So we should delete them? OR do you want to keep them for testing?